### PR TITLE
Allow user to choose userVisible value

### PR DESCRIPTION
**Pull request by nvlbg**
This should allow for an app to choose if the user can request a sync from settings.

### CODE REVIEW

**Comment by ferrannp** [comment]
Hey this looks good! Gonna try to test it later today. Thank you!

**Comment by nvlbg** [comment]
Just fyi, I have another change ready that depends on this one, but I'm not sure if I can make a new pull request that depends on another pull requests, so I'll wait until this one is landed.

File: README.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ The default timeout for your Headless JS task is 5 minutes (300000ms). If you wa
 <string name="rnsb_default_timeout" translatable="false">600000</string>
 ```
 
+### User visible
+
+By default the user will not be able to manually request a sync from the settings. If you want the user to have the option to request a sync from the settings, you have to override `strings.xml` again:
```
Comment (ferrannp) [inline README.md]: I think we need to fix the description. For what I see, this is the difference: https://stackoverflow.com/a/25960728/2277631. So basically, when we set it to `true`, we allow the user to disable/enable the sync functionality using the Account settings. Can we change the description to:

> By default the user will not be able to manually enable/disable syncs through Account settings. If you want the user to have this option, you need to override `strings.xml`:

File: README.md [hunk 2]
```diff
@@ -75,6 +75,15 @@ The default timeout for your Headless JS task is 5 minutes (300000ms). If you wa
...
+
+```xml
+<!-- Allows user to manually request a sync from settings -->
```
Comment (ferrannp) [inline README.md]: We would change this comment to:

> Allows the user to enable/disable the sync functionality through the Account settings.

**Pull request merged**